package.json

```diff
@@ -21,18 +21,19 @@
     "webpack-dev-server": "^3.8.1"
   },
   "dependencies": {
+    "@emotion/core": "^10.0.28",
     "@material-ui/core": "^4.11.0",
     "@material-ui/icons": "^4.5.1",
+    "@material-ui/lab": "^4.0.0-alpha.58",
     "@types/react-beautiful-dnd": "^13.0.0",
+    "emotion": "^10.0.27",
+    "emotion-theming": "^10.0.27",
```

Comment:
There's a bit of noise here (I guess running npm install will re-organize everything?) - but all that's actually been added is the `@material-ui/lab` package to get the alert component